### PR TITLE
BC-2114 - migrate to stylelint 14

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

https://ticketsystem.dbildungscloud.de/browse/BC-2114
https://stylelint.io/migration-guide/to-14/

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/atoms/vRoomEmptyAvatar.vue [hunk 1]
```diff
@@ -56,28 +56,28 @@ export default {
...
-	border: 2px solid;
-	border-color: var(--color-secondary);
+  border: 2px solid;
+  border-color: var(--color-secondary);
 }
 
 .avatar-component-empty {
-	/* stylelint-disable-next-line sh-waqar/declaration-use-variable */
-	border-radius: 1em;
+  /* stylelint-disable-next-line sh-waqar/declaration-use-variable */
```
Comment (MartinSchuhmacher) [inline src/components/atoms/vRoomEmptyAvatar.vue:69]: is the indentation on purpose? its smaller then before

File: package.json [hunk 2]
```diff
@@ -137,12 +138,12 @@
 		"sharp": "^0.28.1",
 		"start-server-and-test": "^1.14.0",
 		"style-loader": "^2.0.0",
-		"stylelint": "^13.12.0",
+		"stylelint": "^14.9.1",
 		"stylelint-config-prettier": "^9.0.3",
-		"stylelint-config-recess-order": "^2.2.0",
-		"stylelint-config-standard": "^22.0.0",
-		"stylelint-declaration-use-variable": "^1.7.3",
```
Comment (MartinSchuhmacher) [inline package.json:144]: is the declaration to use variables provided by the standard vue config?

**Review by MartinSchuhmacher — COMMENTED** [review]
also the github linter does not seem happy with some places of the code :)

File: package.json (hunk 2 above)
Comment (odalys-dataport) [reply package.json:144]: `stylelint-declaration-use-variable` is not actively maintained and recommend the new package `stylelint-declaration-strict-value`

File: src/components/atoms/vRoomEmptyAvatar.vue (hunk 1 above)
Comment (odalys-dataport) [reply src/components/atoms/vRoomEmptyAvatar.vue:69]: No I didn't realize git would run lint:fix. The configurations weren't adjusted yet and are still not fully functional, but I can't find the solution

**Pull request closed without merging**